pkg/config/config.go

```diff
@@ -44,6 +44,14 @@ type Config struct {
 	// only the service model name, but also the iface name to `PipesAPI`. See
 	// https://github.com/aws/aws-sdk-go/blob/main/service/pipes/pipesiface/interface.go#L62
 	SDKNames SDKNames `json:"sdk_names"`
+	// ControllerName lets you specify a service alias override. This can be
+	// used to only override the controller name exposed to the user. Meaning that
+	// it will only change the module name, the import paths, and the controller
+	// name. This is useful when the service identifier is not very user friendly
+	// and you want to expose a more user friendly name to the user. e.g docdb ->
+	// documentdb.
+	// This will also change the helm chart and image names.
+	ControllerName string `json:"controller_name,omitempty"`
```

Comment:
So far this is working very well with `efs-controller` and `documentdb-controller`